contracts/Uniswap/UniswapOracle.sol

```diff
@@ -0,0 +1,102 @@
+pragma solidity ^0.6.6;
+pragma experimental ABIEncoderV2;
+
+import "./UniswapLib.sol";
+
+/**
+ * @notice Uniswap Oracle based on https://github.com/Uniswap/uniswap-v2-periphery/blob/master/contracts/examples/ExampleOracleSimple.sol.
+ * @dev A simple oracle for storing TWAP for 6 assets pairs - ETH, WBTC, ZRX, BAT, REP, DAI.
+ * @author Compound Labs, Inc.
+ */
+contract UniswapOracle {
```

Comment:
Is this being used? Should it be removed?

Reply:
Yes removing it